lua/playerTracking.lua

```diff
@@ -247,30 +247,34 @@ remote.add_interface("playerManager", {
 			player.tag = tag
 			
 			-- Clear old inventories
-			player.get_inventory(defines.inventory.player_quickbar).clear()
-			player.get_inventory(defines.inventory.player_guns).clear()
-			player.get_inventory(defines.inventory.player_ammo).clear()
-			player.get_inventory(defines.inventory.player_tools).clear()
-			player.get_inventory(defines.inventory.player_trash).clear()
-			player.get_inventory(defines.inventory.player_main).clear()
+			--player_quickbar no longer an inventory
+			--player.get_inventory(defines.inventory.player_quickbar).clear()
+			player.get_inventory(defines.inventory.character_guns).clear()
+			player.get_inventory(defines.inventory.character_ammo).clear()
+			-- pickaxe no longer exists
+			--player.get_inventory(defines.inventory.player_tools).clear()
```

Comment:
Unsure why this was commented instead of deleted

Reply:
Just erring on the side of caution. Can delete instead of comment out in future - no problem.